x-pack/plugins/security_solution/public/timelines/components/create_field_button/index.tsx

```diff
@@ -0,0 +1,135 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useCallback, useEffect, useMemo, useState } from 'react';
+import { EuiButton } from '@elastic/eui';
+import styled from 'styled-components';
+
+import { useDispatch } from 'react-redux';
+import { IndexPattern, IndexPatternField } from '../../../../../../../src/plugins/data/public';
```

Comment:
This is using deprecated fields and is out of sync with `main` now:
https://github.com/elastic/kibana/blob/main/x-pack/plugins/security_solution/public/timelines/components/create_field_button/index.tsx#L13

please, let's do a follow up where we sync these and remove the deprecated methods.

You can see hopefully in your IDE the strikethroughs which indicate that this is deprecated and we don't want to reintroduce these.